architecture/architecture.adoc

```diff
@@ -17,13 +17,15 @@ becomes available.
 
 include::modules/architecture-overview.adoc[leveloffset=+1]
 
+include::modules/architecture-components.adoc[leveloffset=+2]
+
 include::modules/installation-options.adoc[leveloffset=+2]
 
-include::modules/update-service-overview.adoc[leveloffset=+2]
+include::modules/update-service-overview.adoc[leveloffset=+3]
 
-include::modules/node-management.adoc[leveloffset=+2]
+include::modules/node-management.adoc[leveloffset=+3]
 
-include::modules/node-types.adoc[leveloffset=+2]
+include::modules/node-types.adoc[leveloffset=+3]
```

Comment:
I attempted to align these offsets w/ the heading level of the content they contain.  Not sure if that is appropriate or not.